marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/CallbackGeneratorTest.java

```diff
@@ -76,20 +96,36 @@ public static void setup() {
        
     }
 
-    @Test(expected = Test.None.class)
+    @Test
     public void forArgsGeneratorTest() {
-        
+        List<Double> outputValues = new ArrayList<Double>();
+        class Output {
```

Comment:
Is the Output class necessary?  or could we just assign inputValues,size() to a final variable?